draft-ietf-quic-ack-frequency.md

```diff
@@ -145,7 +145,7 @@ endpoint performance in the following ways:
 - Sending UDP datagrams is very CPU intensive on some platforms. A data
   receiver can decrease its CPU usage by reducing the number of
   acknowledgement-only packets sent. Experience shows that this
-  reduction can be critical for high bandwidth connections.
+  reduction can be critical for high-rate connections.
```

Comment:
I'm unsure what this means, but I can see why bandwidth may not be exactly correct.  What about "high packet rate connections."?

Reply:
packet rate is fine also